sorc/ncep_post.fd/MDLFLD.f

```diff
@@ -630,6 +640,7 @@ SUBROUTINE MDLFLD
                 DBZC(I,J,L)=CUREFL(I,J)
                ENDIF                   !-- End IF (CUREFL_S(I,J) > 0.)
 
+             IF(T(I,J,L)<spval.and.Q(I,J,L)<spval) THEN
```

Comment:
add check pmid in line 643?
@JesseMeng-NOAA

Reply:
will do

Reply:
done